Code/Editor/ViewportManipulatorController.h

```diff
@@ -46,5 +48,8 @@ namespace SandboxEditor
         AZStd::unordered_map<AzToolsFramework::ViewportInteraction::MouseButton, ClickEvent> m_pendingDoubleClicks;
 
         AZ::ScriptTimePoint m_currentTime;
+        
+        bool m_captureStart;
```

Comment:
update: It's probably a good idea to initialize this to true looking at the code

```
bool m_captureStart = true;
```

Looking at the code I'm a bit concerned `m_virtualNormalizedPosition` could get an invalid value? I think the calculated `screenPoint` might be wrong when there isn't a mouse down event. We'll just be adding a delta increment to `m_virtualNormalizedPosition` before it has a good starting value... 

Maybe `m_virtualNormalizedPosition` should be optional, and we only set it in the Mouse Down event, and then if it has a value, we can then add the `position->m_normalizedPositionDelta` value to it... and only set that to `screenPoint` when the mouse is held (the optional has a value). I think that will be safer...

We just have to be careful to realize moving the mouse will be setting this value even when the mouse button isn't held (say when moving the mouse and picking entities)